### ISSUE TITLE
Error on tray icon right click

### ISSUE THREAD

**Issue by ulcuber** [suurjaak/InputScope#18]
```
Traceback (most recent call last):
  File "/home/victor/.local/lib/python3.9/site-packages/inputscope/main.py", line 235, in OnOpenMenu
    item_session_stop.Enable(bool(activename))
wx._core.wxAssertionError: C++ assertion ""m_menuItem"" failed at /tmp/pip-install-pftclj74/wxpython_1cb442c7599845deaf76319b3bc95202/ext/wxWidgets/src/gtk/menu.cpp(718) in Enable(): invalid menu item
```

**Comment by suurjaak** [comment]
I think I found the problem, but I cannot test it myself, because wx tray icons do not work in Ubuntu.

Can you try with this? [InputScope-1.6.dev15-py2.py3-none-any.zip](https://github.com/suurjaak/InputScope/files/9079386/InputScope-1.6.dev15-py2.py3-none-any.zip)

It's a zipped Python package, unzip it and run `pip install InputScope-1.6.dev15-py2.py3-none-any.whl`.

This should also fix #16 and #17.

**Comment by ulcuber** [comment]
@suurjaak 
Icon context menu opens properly now
#17 fixed. Events are saved now
#16 fixed too

**Comment by suurjaak** [comment]
Most excellent.

I will keep this and the other issues open until the next version is released.

**Comment by suurjaak** [comment]
Released v1.6.
